packages/components/src/Description/Value/Value.tsx

```diff
@@ -12,10 +15,76 @@ export type ValueProps = Pick<
    * Заглушка, отображающаяся, если нет значения
    */
   stub?: ReactNode;
+
+  /**
+   * Добавляет возможность копирования
+   */
+  canCopy?: boolean;
+
+  /**
+   * Позиционирует иконку "копировать" (слева / справа от текста)
+   */
+  copyPosition?: 'left' | 'right';
 };
 
-export const Value = ({ children, stub, ...props }: ValueProps) => {
+export const Value = ({
+  canCopy,
+  copyPosition = 'right',
+  children,
+  stub,
+  ...props
+}: ValueProps) => {
+  const [isCopied, setCopied] = useState(false);
   const { emptySymbol } = useContext(ConfigContext);
 
-  return <Typography {...props}>{children ?? stub ?? emptySymbol}</Typography>;
+  const resultChildren = children ?? stub ?? emptySymbol;
+
+  if (!canCopy) {
+    return <Typography children={resultChildren} {...props} />;
+  }
+
+  const handleClick = () => {
+    setCopied(true);
+    navigator.clipboard.writeText(children?.toString() || '');
+  };
+
+  const handleMouseLeave = () => {
+    if (isCopied) {
+      // Ставим таймер чтобы тултип успел скрыться
+      // В ином случае пользователь увидит изменение текста
+      setTimeout(() => {
+        setCopied(false);
+      }, 100);
+    }
+  };
+
+  const tooltipText = isCopied ? 'Скопировано' : 'Скопировать';
+
+  return (
+    <Tooltip
+      onClick={handleClick}
+      onMouseLeave={handleMouseLeave}
+      withoutContainer={false}
+      placement="bottom"
+      title={tooltipText}
+    >
+      <StyledText {...props}>
```

Comment:
```suggestion
      <StyledTypography {...props}>
```

В данном случае думаю лучше назвать компонент так, тогда становится понятно что за пропсы он может принимать